config/00-main.yaml

```diff
@@ -14,5 +14,5 @@ database:
   dbname: wikilabels
 
 oauth:
-  mw_uri: https://www.mediawiki.org/w/index.php
+  mw_uri: https://www.meta.wikimedia.org/w/index.php
```

Comment:
Besides that I'm not sure if we should do this at all, www.meta.wikimedia.org is wrong, you need to omit www.